.eslintignore

```diff
@@ -2,3 +2,4 @@ es/
 lib/
 *.d.ts
 node_modules/
+examples/
```

Comment:
Why would we not lint the examples as well?

Reply:
this will cause problems with the eslint packages inside `examples/react-example/node_modules` and tbh I don't think its important to lint example files